src/website/webview/urls.py

```diff
@@ -1,8 +1,14 @@
-from django.urls import path
+from django.urls import path, register_converter
 
-from webview.views import HomeView
+from webview.views import HomeView, NixpkgsIssueView, NixpkgsIssueListView
+from webview.converters import NixpkgsIssueConverter
 
 app_name = "webview"
+
+register_converter(NixpkgsIssueConverter, "nixpkgs")
+
 urlpatterns = [
     path("", HomeView.as_view(), name="home"),
+    path("issue/<nixpkgs:id>", NixpkgsIssueView.as_view(), name="issue_detail"),
```

Comment:
ideally, it should be `/<nixpkgs:id>` directly, no need for an additional subpath, leave it for the rest.
the central feature is here

Reply:
No, this pollutes the namespace. Leave me that piece of the bikeshed.